composer.json

```diff
@@ -21,6 +21,12 @@
 		"symfony/console": "3.4.16 || 4.1.5"
 	},
 	"require-dev": {
+		"ext-curl": "*",
+		"ext-intl": "*",
+		"ext-mbstring": "*",
+		"ext-mysqli": "*",
+		"ext-openssl": "*",
+		"ext-soap": "*",
```

Comment:
Not all of those are required by PHPStan. Some of them are in appveyor.yml so that Composer can work and for example mbstring is there because of PHPUnit.

Try to remove curl, mbstring and openssl.